test/iso/component-param.js

```diff
@@ -0,0 +1,179 @@
+import sinon from 'sinon';
+import {expect} from 'chai';
+import {
+  DefaultAndRequiredParam,
+  ExtraParamPassInChild,
+  ParamChild,
+  ParamParentApp,
+  NonPrimitiveTypeParamClass,
+  NonPrimitiveTypeParamString,
+  ShouldComponentUpdateParamsApp,
+} from '../fixtures/params-app';
+import {nextAnimationFrame, compactHtml} from '../utils';
+
+customElements.define(`param-child`, ParamChild);
+customElements.define(`param-parent-app`, ParamParentApp);
+customElements.define(`param-required-and-default-app`, DefaultAndRequiredParam);
+customElements.define(`extra-param-pass-in-child`, ExtraParamPassInChild);
+customElements.define(`non-primitive-type-param-class`, NonPrimitiveTypeParamClass);
+customElements.define(`non-primitive-type-param-string`, NonPrimitiveTypeParamString);
+customElements.define(`should-component-update-params-app`, ShouldComponentUpdateParamsApp);
+
+describe(`panel-params`, () => {
```

Comment:
```suggestion
describe(`Panel params`, function() {
```
change arrow functions to normal functions for mocha callbacks throughout